### PR TITLE
Added the abbreviation of Test Driven Development where first mentioned

### PR DESCRIPTION
**Pull request by khunhour**
added the abbreviation where Test Driven Development is first mentioned, so people will not get confused to what TDD later on in the lesson.

 - [x] You have read [The Odin Projects Contributing Guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md).
 - [x] The title summarizes the change and where it happened, for example: "Fixes punctuation in Clean Code lesson".
 - [x] If the PR is related to an open issue, use a [relevant keyword](https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests) and reference it with the `#` sign and the issue number.
 - [x] If changes are requested, please make the changes in a timely manner. After you submit the changes, request another review from the maintainer (top right).

#### 1. Describe the changes made and include why they are necessary or important:

added an abbreviation of Test Driven Development where it is first mentioned, so people will not get confused when they see it later on in the lesson.

#### 2. Related Issue

Closes #XXXXX

### CODE REVIEW

File: javascript/testing/testing-1.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ### Introduction
-Test Driven Development is a _big_ deal in the modern development landscape.  This is a concept that we introduced way back in our Fundamentals section with our JavaScript Exercises.  The main idea is simply that you start working on your code by writing automated tests _before_ writing the code that is being tested.  There are tons of benefits to working like this, all of which will be discussed in the resources below.
+Test Driven Development (TDD) is a _big_ deal in the modern development landscape.  This is a concept that we introduced way back in our Fundamentals section with our JavaScript Exercises.  The main idea is simply that you start working on your code by writing automated tests _before_ writing the code that is being tested.  There are tons of benefits to working like this, all of which will be discussed in the resources below.
```
Comment (zachmmeyer) [inline javascript/testing/testing-1.md:2]:
```suggestion
Test Driven Development, or TDD for short, is a _big_ deal in the modern development landscape.  This is a concept that we introduced way back in our Fundamentals section with our JavaScript Exercises.  The main idea is simply that you start working on your code by writing automated tests _before_ writing the code that is being tested.  There are tons of benefits to working like this, all of which will be discussed in the resources below.
```

**Review by zachmmeyer — CHANGES_REQUESTED** [review]
Definitely a good idea to introduce the abbreviation. I have a minor suggestion here.

**Comment by khunhour** [comment]
Right, that is much clearer. thanks for the suggestion.

**Review by zachmmeyer — APPROVED** [review]

**Comment by zachmmeyer** [comment]
Thank you for contributing!!

**Pull request merged**